src/window_gui.h

```diff
@@ -755,6 +758,13 @@ struct Window : ZeroedMemoryAllocator {
 	 */
 	virtual bool OnVehicleSelect(const struct Vehicle *v) { return false; }
 
+	/**
+	 * The user clicked on a vehicle while HT_VEHICLE has been set.
+	 * @param v clicked vehicle. It is guaranteed to be v->IsPrimaryVehicle() == true
```

Comment:
Doxygen headers should be written in plain English, with no code included.

Reply:
This is copied from the `bool OnVehicleSelect(const struct Vehicle *v);` function immediately above.

But anyway what we actually want is a precondition, so I've changed it and the function immediately above.